modules/core/src/main/java/io/fluo/core/client/FluoClientImpl.java

```diff
@@ -66,7 +66,7 @@ public FluoClientImpl(FluoConfiguration config) {
   @Override
   public LoaderExecutor newLoaderExecutor() {
     try {
-      return new LoaderExecutorImpl(config, env);
+      return new LoaderExecutorAsyncImpl(config, env);
```

Comment:
not really... while working on this I had two implementations for a bit (the old sync one and the new async one)... then I deleted the old sync classes